modules/server_processes_manager/php/server_processes_manager.class.inc

```diff
@@ -114,9 +111,17 @@ class Server_Processes_Manager extends \NDB_Menu_Filter
         $deps    = parent::getJSDependencies();
         return array_merge(
             $deps,
-            array($baseURL . "/server_processes_manager/js/columnFormatter.js")
+            [
+             $baseURL . "/server_processes_manager/js/columnFormatter.js",
```

Comment:
You can remove column formatter as a JS dependency

Reply:
you just have to delete it in `jsx/` and then it should disappear from `js/` IFRC. You do not need to commit anything in the `js/` directory to major.